### PR TITLE
feat(quadlet): log option handling

### PR DESCRIPTION
**Pull request by breca**
I found that Quadlet didn't currently have support for log options. This merge allows Quadlet to handle log options and correctly pass those values through to `podman run` for Container and Kube types.

Syntactically consistent with existing parameters:

```ini
[Container]
Image=localhost/imagename
LogOpt=path=/var/log/container/mycontainer.json
LogOpt=size=10mb
```

#### Does this PR introduce a user-facing change?

```release-note
Added support for log driver options to Quadlet (https://docs.podman.io/en/latest/markdown/podman-systemd.unit.5.html#logopt)
```

### CODE REVIEW

File: docs/source/markdown/podman-systemd.unit.5.md [hunk 1]
```diff
@@ -557,6 +558,11 @@ This key can be listed multiple times.
 Set the log-driver used by Podman when running the container.
 Equivalent to the Podman `--log-driver` option.
 
+### `LogOpt=`
+
+Set the log-opt (logging options) used by Podman when running the container.
+Equivalent to the Podman `--log-opt` option.
```
Comment (ygalblum) [inline docs/source/markdown/podman-systemd.unit.5.md:564]: Please add `This key can be listed multiple times.`

**Review by ygalblum — COMMENTED** [review]
Thanks for the PR. Other than my small nit on the documentation all looks great

File: docs/source/markdown/podman-systemd.unit.5.md (hunk 1 above)
Comment (breca) [reply docs/source/markdown/podman-systemd.unit.5.md:564]: Added

**Comment by breca** [comment]
> Thanks for the PR. Other than my small nit on the documentation all looks great

Updated - cheers

**Comment by ygalblum** [comment]
LGTM

**Review by ygalblum — APPROVED** [review]
LGTM

**Review by vrothberg — APPROVED** [review]
/lgtm

**Pull request merged**